src/Server/Grabber/Grabber.ts

```diff
@@ -11,6 +11,24 @@ export class Grabber {
 
     }
 
+    public static getCourse(course: string): Promise<string[]> {
```

Comment:
Change to other data structure instead a string. Maybe like
``` typescript
interface CourseInfo{
  subject: string;
  course_id: string;
  section_id: string;
}
```

Reply:
I will keep subject and course_id but leave section_id as currently fetching course information doesn't need that yet. We may want a data structure specifically for section